hummingbot/client/command/create_command.py

```diff
@@ -41,10 +41,15 @@ async def prompt_for_configuration(self,  # type: HummingbotApplication
         self.placeholder_mode = True
         self.app.hide_input = True
         required_exchanges.clear()
+        # add self.stop_config to know if cancel is triggered
+        self.stop_config = None
```

Comment:
It's not very clear here if this attribute is a boolean, you can just initiate it to `False` (instead of `None`) if it's gonna be only either True or False. And perhaps use a more indicative name like `to_stop_config` or `to_exit_config`.